test/python/primitives/test_estimator.py

```diff
@@ -350,7 +349,7 @@ class TestObservableValidation(QiskitTestCase):
         ("IXYZ", (SparsePauliOp("IXYZ"),)),
         (Pauli("IXYZ"), (SparsePauliOp("IXYZ"),)),
         (SparsePauliOp("IXYZ"), (SparsePauliOp("IXYZ"),)),
-        (PauliSumOp(SparsePauliOp("IXYZ")), (SparsePauliOp("IXYZ"),)),
+        (SparsePauliOp(SparsePauliOp("IXYZ")), (SparsePauliOp("IXYZ"),)),
```

Comment:
You don't need to test `SparsePauliOp(SparsePauliOp(...))`. The test of `SparsePauliOp(...)` is enough.